components/service/glean/src/main/java/mozilla/components/service/glean/testing/GleanTestRule.kt

```diff
@@ -24,9 +24,13 @@ import org.junit.runner.Description
  * @get:Rule
  * val gleanRule = GleanTestRule(ApplicationProvider.getApplicationContext())
  * ```
+ *
+ * @param context the application context
+ * @param configToUse an optional [Configuration] to initialize the Glean SDK with
  */
 class GleanTestRule(
-    val context: Context
+    val context: Context,
+    val configToUse: Configuration = Configuration()
```

Comment:
I guess this doesn't need public documentation [in the book](https://mozilla.github.io/glean/book/user/testing-metrics.html), as it shouldn't concern the majority of users, right?

Reply:
Correct!